pretty_cron/api.py

```diff
@@ -106,7 +121,17 @@ def _human_list(listy):
 
 
 def _human_week_day(day):
-    return _WEEKDAYS[day]
+    """
+    Cron-tab to weekday
```

Comment:
ditto, vacant docstring, please remove (just because pylint tells you to do something...)